src/pages/Home/index.jsx

```diff
@@ -147,13 +143,13 @@ export default class Home extends Component {
   render() {
     const { rootStore, appStore, categoryStore, match } = this.props;
     const { fixNav } = rootStore;
-    const { homeApps, isLoading, isProgressive } = appStore;
+    const { apps, isLoading, isProgressive } = appStore;
     const categories = categoryStore.categories;
-    const categoryId = match.params.category;
-    const appSearch = match.params.search;
-    const showApps = appSearch || Boolean(categoryId) ? homeApps.slice() : homeApps.slice(0, 3);
+
+    const { category, search } = match.params;
+    const showApps = category || search ? apps : apps.slice(0, 3);
```

Comment:
The home page show newest apps  and category apps, add homeApps variable for storage  newest apps, becase query category apps will change apps data. If remove  homeApps variable, the newest apps show is error.

Reply:
Got, i will check it